manual/sphinx/user_docs/differential_operators.rst

```diff
@@ -153,6 +154,9 @@ The correction factor is then calculated from ``d2x`` using
 
    {{\frac{\partial }{\partial i}}}(\frac{1}{\Delta x}) = -\frac{1}{\Delta x^2} {{\frac{\partial \Delta x}{\partial i}}}
 
+**Note**: There is a separate switch in the :ref:`Laplacian inversion code <sec-laplacian>`,
```

Comment:
Should this include something like "overrides" or mentions that is defaults to the global/mesh setting?